frontend/src/pages/QueuePage.jsx

```
@@ -37,6 +37,7 @@ export default function QueuePage() {
   const moveToPositionMutation = useMoveToPosition()
   const snoozeMutation = useSnooze()
   const unsnoozeMutation = useUnsnooze()
+  const snoozeMutation = useSnooze()
```

Biome (2.3.14)

[error] Shouldn't redeclare 'snoozeMutation'. Consider to delete it or rename it.

'snoozeMutation' is defined here:


(lint/suspicious/noRedeclare)